arm/network/client.go

```diff
@@ -1,5 +1,5 @@
 // Package network implements the Azure ARM Network service API version
-// 2016-06-01.
+// 2016-09-01.
 //
```

Comment:
Did you know whether there were any breaking changes between 2016-07-01 & 2016-09-01?